packages/client-core/src/social/services/ChatService.ts

```diff
@@ -215,6 +215,8 @@ export const accessChatState = () => state
 
 export const useChatState = () => useState(state) as any as typeof state
 
+globalThis.chatState = state
```

Comment:
why does chat state need to be on the global state? can we use the accessChatState hook instead?